lib/developer_portal/app/controllers/developer_portal/admin/applications_controller.rb

```diff
@@ -17,6 +17,8 @@ class DeveloperPortal::Admin::ApplicationsController < ::DeveloperPortal::BaseCo
 
   liquify prefix: 'applications'
 
+  delegate :services, :multiservice?, to: :site_account
```

Comment:
For me personally this delegation is a little confusing. But feel free to ignore if others are ok with that. Maybe it's my lack of experience with this part of the code.